### ISSUE TITLE
null reference issue

### ISSUE THREAD

**Issue by LarkLib** [rubanraj54/vue-bootstrap4-table#4]
I got a null reference issue when global search a string.
I think this line pop it.
VueBootstrap4Table.vue
line 661:
`                    if (typeof value === "undefined") {
`
It's working when changed to:
`                    if (value == null || typeof value === "undefined") {
`

**Comment by rubanraj54** [comment]
Thanks for reporting the issue. @LarkLib 

I've planned for a new release with more features in this weekend.
I will fix the above issue in new release.

**Comment by LarkLib** [comment]
It got a "Promise is undefined" issue in IE 11
May be to run  "npm install --save babel-polyfill" to resolute it.

**Comment by rubanraj54** [comment]
> It got a "Promise is undefined" issue in IE 11
> May be to run "npm install --save babel-polyfill" to resolute it.
> 
> pop this error in my app when run in IE 11, it should be also a compatibility issue.
> 
> SCRIPT1046: Multiple definitions of a property not allowed in strict mode
> [Vue warn]: Unknown custom element: - did you register the component correctly? For recursive components, make sure to provide the "name" option.

is this a new issue or belongs to the "null reference issue"? @LarkLib 

I will do a test in IE11 also. But IMHO, use es6-promise polyfill plugin in your app.

`require('es6-promise').polyfill();`

Anyway I will give a try with polyfill and check how the final bundle looks like.

**Comment by rubanraj54** [comment]
Hey @LarkLib 

I tried with IE 10 and 11. I got only the polyfill error and this is not from the plugin. 

The polyfill error was from axios request to pull the dummy data to populate the table for testing.

If you use axios or promise based plugin in your application, then just install [es6-promise](https://github.com/stefanpenner/es6-promise) and include this `require('es6-promise').polyfill();` code in your main javascript file. This solves the promise issue in IE >= 10.

I didn't get the other error which you mentioned above.
> SCRIPT1046: Multiple definitions of a property not allowed in strict mode 

And the real issue "null reference issue" is fixed in the develop branch and will be included in the next release most probably this weekend.
